### DISCUSSION TITLE
How do I fine control the volume using the keyboard?

### DISCUSSION THREAD

**Discussion by freediverx** [MonitorControl/MonitorControl#876] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When controlling my external display's speaker volume using keyboard controls (made possible by MonitorContol), adjustments are only available in increments of six units, which is not nearly granular enough. This limitation makes keyboard volume control nearly useless, since available settings are always either too quiet or too loud.

I can control volume in increments of one using either the display's built-in hardware control or with MonitorContol's menubar applet slider.

### Steps to reproduce

Tap the volume up and volume down keyboard buttons and observe the incremental changes in the volume indicator.

### Expected behavior

Volume should change in increments of 1 for every button press, rather than in increments of 6.

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey 12.2
- Mac model: 2020 13" Macbook Pro
- MonitorControl version: 4.0.2
- Monitor(s): LG 32BL95U-W 32" 4K Thunderbolt 
- Apple Silicon/M1 (yes or no): no
```

**Comment by the0neyouseek** [comment, marked as answer]
Hi @freediverx

You can use the "Use fine OSD scale for volume" options in the app settings to enable a more granular scale for volume control (or the `Alt`+`Shift`+`Volume Up/Down` combo key)

Feel free to respond if that does not solve your issue.
Have a nice day

**Comment by freediverx** [comment]
Oh wow, I completely overlooked that setting. I'll give it a try...

**Comment by freediverx** [comment]
Works great. Thanks.

**Comment by waydabber** [comment]
You can also use change the volume min-max setting if you want to remap the scale. If you set the max setting to 16 or 32 for example then you can use the full Volume OSD scale but the max volume setting will still be 16 or 32 instead of 100 (16 or 32 corresponds nicely to the chiclets in the OSD overlay).

<img width="842" alt="Screen Shot 2022-01-04 at 18 19 57" src="https://user-images.githubusercontent.com/37590873/148098219-a3e716fa-425a-45fc-8daa-8f302ddb4bea.png">
